### PR TITLE
Recent blocks summary

### LINKED ISSUE #4: Recent block interval, difficulty & hash rate
As like [Etherscan] shows the recent block interval, difficulty, and hash rate (see below), it would be useful if we have the similar thing.

From Etherscan's home page

[Etherscan]: https://etherscan.io/

### PR DESCRIPTION
**Pull request by aucch**
Close #4

### CODE REVIEW

**Comment by aucch** [comment]
I worked done this.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -40,8 +40,34 @@ const IndexPage: React.FC<IndexPageProps> = ({ location }) => {
...
+              interval += +timestamps[i] - +timestamps[i + 1];
+            }
+            interval /= (timestamps.length - 1) * 1000;
+          }
+
+          const difficulties: number[] | null = data && data.blocks
+            ? data.blocks.map(block => block!.difficulty)
+            : null;
+          let difficulty = 0;
+          if (difficulty != null && difficulties) {
```
Comment (longfin) [inline src/pages/index.tsx]: `difficulty != null` check seems to pass always because we assign `0` to it. right?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -40,8 +40,34 @@ const IndexPage: React.FC<IndexPageProps> = ({ location }) => {
...
+            for (let i = 0; i < difficulties.length; i++)
```
Comment (longfin) [inline src/pages/index.tsx]: We can also use [`Array.prototype.reduce()`](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce).

```suggestion
            difficulty = difficulties.reduce((d, sum) => d + sum, 0);
```

File: src/pages/index.tsx (hunk 1 above)
Comment (longfin) [reply src/pages/index.tsx]: I'd heard about it from @aucch verbally that it's needed because TypeScript claims `difficulty` can be `null`.

**Pull request merged**